db/migrate/20211018080221_add_role_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddRoleToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :role, :integer, default:0, null: false
```

Comment:
`default: 0`

Reply:
You didn't add space between `default:` and `0`

Reply:
Integer columns are hard to understand without actual context.

One of the downsides of using an integer column for storing a representation of a string value, like the case of a role column, is that we are going to make it harder for people looking directly at the database table's rows to know what number represents which status. You can have more roles, there is no limit.

Instead of using an integer column, you can use a string one.

Reply:
YOU DIDN'T ADD SPACE BETWEEN `default: 0` !!!!!!!!!!!